### DISCUSSION TITLE
【征集】各类（高清无码）聊天截图

### DISCUSSION THREAD

**Discussion by jwyjohn** [jwyjohn/TJCovidDiary2022#3] [Announcements]
> 太史公曰：天道恢恢，岂不大哉！谈言微中，亦可以解纷。
> 
> 滑稽鸱夷，如脂如韦。敏捷之变，学不失词。淳于索绝，赵国兴师。楚优拒相，寝丘获祠。伟哉方朔，三章纪之。

和古人的信件一样，现代大学生的聊天截图是记录历史的重要载体，有着极高的参考价值。

故而本猫征集此期间的各类聊天截图，和对截图的简短描述；以期经过整理之后，藏之名山。

**Comment by PlusOneZ** [comment]
可以加一个 meme 的分类不🥹

**Reply by jwyjohn** [reply]
加，都可以加

**Comment by Zppai** [comment]
need 马蹄酥
